diff --git a/Android.mk b/Android.mk
index ce4e8cf..a1b8c07 100644
--- a/Android.mk
+++ b/Android.mk
@@ -16,7 +16,7 @@ $(fdroid_root)/$(fdroid_dir)/$(fdroid_apk):
        rm -Rf $(fdroid_build)
        mkdir -p $(fdroid_out)
        ln -sf $(fdroid_out) $(fdroid_build)
-       cd $(fdroid_root)/$(fdroid_dir) && gradle assembleRelease
+       cd $(fdroid_root) && ./gradlew assembleRelease
 
 LOCAL_CERTIFICATE := platform
 LOCAL_PRIVILEGED_MODULE := true
diff --git a/build.gradle b/build.gradle
index 49b0130..2f7a80c 100644
--- a/build.gradle
+++ b/build.gradle
@@ -1,6 +1,8 @@
 buildscript {
     repositories {
-        jcenter()
+        maven {
+                url "https://jcenter.bintray.com"
+              }
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:2.1.3'
