diff --git a/BoardConfig.mk b/BoardConfig.mk
index 4b6d737..95f5297 100644
--- a/BoardConfig.mk
+++ b/BoardConfig.mk
@@ -26,7 +26,7 @@ TARGET_SPECIFIC_HEADER_PATH += device/sony/z3c/include
 BOARD_HARDWARE_CLASS += device/sony/z3c/cmhw
 
 # Kernel properties
-TARGET_KERNEL_CONFIG := cm_shinano_aries_defconfig
+TARGET_KERNEL_CONFIG := cm12_shinano_aries_defconfig
 
 # Partition information
 BOARD_VOLD_MAX_PARTITIONS := 25

